cli/src/semgrep/commands/scan.py

```diff
@@ -365,6 +365,18 @@ def scan_options(func: Callable) -> Callable:
     default=False,
 )
 @optgroup.option("--version", is_flag=True, default=False)
+@optgroup.option(
+    "--x-ls",
+    is_flag=True,
+    default=False,
+    help=(
+        "[INTERNAL] List the selected target files and the skipped target"
```

Comment:
the help here is useless because osemgrep is now handling semgrep scan --help

Reply:
Better to remove it (I've removed all the help=() in this file a month ago).

Reply:
sure, I noticed it was useless after the fact.